backend/src/main/java/com/redhat/cloud/notifications/db/BehaviorGroupResources.java

```diff
@@ -41,6 +41,13 @@ public BehaviorGroup createDefault(BehaviorGroup behaviorGroup) {
 
     @Transactional
     BehaviorGroup create(String accountId, BehaviorGroup behaviorGroup, boolean isDefaultBehaviorGroup) {
+        List<BehaviorGroup> behaviorGroupsSameDisplayNameAlreadyPresent = entityManager.createNamedQuery("findByDisplayNameAndAccountId", BehaviorGroup.class)
+                .setParameter("displayName", behaviorGroup.getDisplayName())
+                .setParameter("accountId", behaviorGroup.getAccountId())
```

Comment:
```suggestion
                .setParameter("accountId", accountId)
```
(the behavior group `accountId` field should always be `null` at this point of the method execution)

Reply:
done